polaris-react/src/components/Choice/Choice.scss

```diff
@@ -5,6 +5,10 @@
   justify-content: flex-start;
   padding: var(--p-space-1) 0;
   cursor: pointer;
+
+  @media #{$p-breakpoints-sm-down} {
+    width: 100%;
```

Comment:
This could be `display: flex` instead of `width`

You could refactor this to be mobile first by setting `display: flex` on `.Choice` and then updating it to `display: inline-flex` at `p-breakpoints-sm-up`

Reply:
Can you provide more context on why that is a better solution here? :)